lnbits/app.py

```diff
@@ -579,6 +579,12 @@ def configure_logger() -> None:
     logging.getLogger("uvicorn.error").handlers = [InterceptHandler()]
     logging.getLogger("uvicorn.error").propagate = False
 
+    logging.getLogger("sqlalchemy").handlers = [InterceptHandler()]
+    logging.getLogger("sqlalchemy.engine.base").handlers = [InterceptHandler()]
+    logging.getLogger("sqlalchemy.engine.base").propagate = False
+    logging.getLogger("sqlalchemy.engine.base.Engine").handlers = [InterceptHandler()]
+    logging.getLogger("sqlalchemy.engine.base.Engine").propagate = False
+
```

Comment:
why is this required?

Reply:
to remove duplications and fit it in better with the other logs.
![screenshot-1711005251](https://github.com/lnbits/lnbits/assets/1743657/68c5d327-25d9-4d66-91d5-e6296d7bd99b)

Reply:
compare to the above in the original screenshot, there were duplications and no formatting